nextflow_schema.json

```diff
@@ -96,26 +96,26 @@
             "description": "These parameters influence the database to be used in classifying the taxonomy.",
             "default": "",
             "properties": {
-                "taxa_classification_mmseqs_databases_localpath": {
+                "taxa_classification_mmseqs_db": {
                     "description": "Specify a path to MMseqs2-formatted database.",
                     "help_text": "Specify a path to a database that is prepared in MMseqs2 format as detailed in the [documentation](https://mmseqs.com/latest/userguide.pdf).",
-                    "fa_icon": "fab fa-stackpath"
+                    "fa_icon": "fas fa-database"
```

Comment:
Line above is missing the `modifies tool parameter(s)` bit

Reply:
Also check throughout everywhere if we are doing the tool naming by the process name or the tool itself (e.g. `MMSEQS_TAXONOMY` vs `mmseqs taxonomy`. The latter makes more sense from  a user PoV if tey go back to check the docs of the tool itself

Reply:
(or just the 'official' tool name... maybe that's what we went with...)

Reply:
Yep, it's the tool name in all cases :)

Reply:
I checked again, this doesn't modify a tool parameter. Rather in the pipeline logic decided whether a DB will be downloaded or the DB from this param being used.

Reply:
Okay added the correct environment variable anyway ;)